Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,26 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            for (int i = 0; i < 2; i++)
```

Comment:
Looks good but yuo could have used a function and cal it twice instead of a loop. Would read better.